### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by rozhi-98**
I have added a media query for desktop screens, 768px and higher as required in the project requirements. I have styled the desktop version according to the figma template I had chosen at the beginning of the project. I have tried to follow the same style as much as possible which required me to change some content of photos and and texts to match the desktop style of the figma template. The website is completely responsive and I have tried to follow best practices.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :root {
```
Comment (HAMMAS-SALEEM) [inline style.css:1]: ### Your Navbar

image

### Figma Navbar

image


- [ ] Your navbar doesn't have the right color as shown in the Figma image above. Kindly make your navbar background color white to follow the Figma template :blush:

Comment (HAMMAS-SALEEM) [inline style.css:1]: ### Your Homepage Design
your design

### Figma Design
image

- [ ] Your design is not matching with the Figma template. Kindly compare your design with the figma template. I have highlighted those parts with red boxes. Such as your hero title and paragraph and your icons are not at proper place :thinking: kindly make your design look like Figma template to make it professional :blush:

Comment (HAMMAS-SALEEM) [inline style.css:1]: ### Your Boxes
image

### Figma Boxes
image

- [ ] Your boxes are not equal size as your can compare it with the Figma template :blush: and make it look like Figma template :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @rozhi-98 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following some requirements
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @rozhi-98 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**